libs/damap/src/lib/widgets/export-warning-dialog/export-warning-dialog.html

```diff
@@ -10,7 +10,7 @@ <h1 mat-dialog-title>{{'dialog.export.title' | translate}}</h1>
             <mat-option
               *ngFor="let id of dmpTemplate | keyvalue"
               [value]="id.key">
-              {{id.key}}<br />
+              {{translateTemplatePrefixEnum + id.key | translate}}<br />
```

Comment:
Do we want to translate the whole thing or just the `translateTemplatePrefixEnum`? Afaik, the organizations should not be translated.